src/site/content/en/live/index.njk

```diff
@@ -39,7 +39,7 @@ date: 2020-06-31
         </p>
 
         <div class="w-event-buttons">
-          <a class="w-button w-button--primary" href="#subscribe">
+          <a class="w-button w-button--primary gc-analytics-event" data-category="web.dev" data-label="view-subscribe" data-action="click" href="#subscribe">
```

Comment:
Let's go with a different name for this label to disambiguate it from the FAB. It's possible to sort by page type in analytics but I think a different label name will be quicker. Might call it 'scroll-to-subscribe' or something?